masonry/src/widget/prose.rs

```diff
@@ -190,8 +190,17 @@ impl Widget for Prose {
         }
     }
 
-    fn on_access_event(&mut self, _ctx: &mut EventCtx, _event: &AccessEvent) {
-        // TODO - Handle accesskit::Action::SetTextSelection
+    fn on_access_event(&mut self, ctx: &mut EventCtx, event: &AccessEvent) {
+        if event.target == ctx.widget_id() {
```

Comment:
You can remove that condition. Both Prose and Textbox are leaf widgets: if they get an accessibility event, we know for sure they're the target.

(Also `AccessEvent::target` was removed in a recent PR, so you'll need to anyway to merge this.)